Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -318,14 +351,19 @@ class SentryUserFeedbackForm: UIViewController {
         return textView
     }()
     
-    lazy var screenshotImageView = UIImageView()
+    lazy var screenshotImageView = {
```

Comment:
`l`: Explicitly defining the type of screenshotImageView could improve compile time.

```suggestion
    lazy var screenshotImageView: UIImageView = {
```

Maybe possible to measure using these `OTHER_SWIFT_FLAGS`: 
```
-Xfrontend -warn-long-function-bodies=50 -Xfrontend -warn-long-expression-type-checking=50
```

Maybe helpful: https://github.com/fastred/Optimizing-Swift-Build-Times

Reply:
I used to recommend this more but haven't noticed much difference in modern times. Just to be sure, I checked the build time analysis with no explicit type annotations (so, the file as it is today) and with _everything_ explicitly annotated:

|before|after|
|---|---|
|<img width="910" alt="image" src="https://github.com/user-attachments/assets/009f6cbe-9553-4096-b828-67b5a18bd992" />|<img width="929" alt="image" src="https://github.com/user-attachments/assets/3a21ffc8-2fe1-4645-81ab-17a256e475a8" />|

So about 113 additions of types only saved about 100 ms. I'm gonna pass on this one!